lib/addons/jupyterhub/index.ts

```diff
@@ -14,6 +14,26 @@ import * as semver from 'semver';
 import { EbsCsiDriverAddOn } from "../ebs-csi-driver";
 import { EfsCsiDriverAddOn } from "../efs-csi-driver";
 
+/**
+ * Configuration options for exposing the JupyterHub proxy
+ */
+export enum jupyterHubServiceType {
```

Comment:
Enum/classes/interface/types - start with upper case (affects code elsewhere and docs)

Reply:
> Enum/classes/interface/types - start with upper case (affects code elsewhere and docs)

@youngjeong46, looks like I am the one who submitted the PR and you are the reviewer. please disregard, I will address.